Shared/Supporting Files/Models/Sample.swift

```diff
@@ -41,6 +41,12 @@ protocol Sample {
 // MARK: Computed Variables
 
 extension Sample {
+    /// The URL to a sample's sub-directory on GitHub main branch.
+    var githubURL: URL {
```

Comment:
It is GitHub, not Github: `githubURL` -> `gitHubURL`

Reply:
🤕 fixed